cmake/Modules/FindTinyXML2.cmake

```diff
@@ -13,7 +13,26 @@ if(TinyXML2_FOUND)
       set(TINYXML2_LIBRARY tinyxml2)
     elseif(TARGET tinyxml2::tinyxml2)
       set(TINYXML2_LIBRARY tinyxml2::tinyxml2)
+    elseif(TinyXML2_FIND_REQUIRED)
+      message(FATAL_WARNING "Unable to determine target for TinyXML2")
```

Comment:
What is `FATAL_WARNING` here? That doesn't seem to be a valid mode: https://cmake.org/cmake/help/v3.17/command/message.html